typings/index.d.ts

```diff
@@ -1022,7 +1021,8 @@ export class Interaction extends Base {
   public type: InteractionType;
   public user: User;
   public version: number;
-  public inGuild(): this is this & { guildId: Snowflake; member: GuildMember | APIInteractionGuildMember };
+  public inGuild(bot?: boolean): this is this & { guildId: Snowflake; member: GuildMember | APIInteractionGuildMember };
+  public inGuild(bot: true): this is this & { guildId: Snowflake; member: GuildMember };
```

Comment:
These need to be inversed

Reply:
I'm confused, if we are checking for the cached guild with the bot and it finds the guild isn't this correct?

Reply:
he's just saying the bot?: boolean overload should always be last